content/en/docs/profiles.md

```diff
@@ -0,0 +1,132 @@
+
+<sup>†</sup>: Only included for certificates with RSA public keys.
+
+</div>
+<div class="boxed">
+
+## tlsserver
+
+The tlsserver profile is a new profile which updates several of these validation and certificate properties in order to reflect the latest recommendations from the CA/Browser Forum Baseline Requirements, as well as general trends within the WebPKI community. We recommend selecting this profile for subscribers who want smaller certificates and who fully embrace automation.
+
+The pending authorization lifetime has been reduced to further encourage automation: fully automated systems can complete a validation challenge within seconds, so a lifetime of just one hour is more than enough. The authorization reuse period has been reduced to seven hours. This is because the Baseline Requirements require that we re-check Certificate Authority Authorization (CAA) after eight hours, so limiting the reuse period means that we don't have to perform rechecks. The order lifetime has been reduced to the sum of two authorization lifetimes, because there is little purpose to having an order that outlives the authorizations it depends on.
+
+The issued certificate no longer contains any of the fields discussed above. The Common Name has been omitted, as it is redundant with the Subject Alternative Names and is marked as NOT RECOMMENDED by the Baseline Requirements. The Key Encipherment key usage is omitted because it is only relevant when using non-forward-secret TLS cipher suites, which have been removed by all major browsers due to the importance of forward-secrecy. The TLS Client Auth extended key usage is omitted to comply with upcoming root program requirements that require "single-purpose" (i.e. single EKU) certificates. And the Subject Key ID extension is omitted because it serves no purpose in end-entity certificates and is NOT RECOMMENDED by the Baseline Requirements.
+
+| Property | Value |
+| -------- | ----- |
+| [Pending Authorization Lifetime](#pending-authorization-lifetime) | 1 hour |
+| [Authorization Reuse Period](#authorization-reuse-period) | 7 hours |
+| [Order Lifetime](#order-lifetime) | 8 hours |
+| [Certificate Common Name](#certificate-common-name) | No |
+| [Key Encipherment KU](#key-encipherment-key-usage) | No |
+| [TLS Client Auth EKU](#tls-client-authentication-extended-key-usage) | No |
+| [Subject Key ID](#subject-key-identifier-extension) | No |
+| [Validity Period](#validity-period) | 90 days |
+
+</div>
+<div class="boxed">
+
+## shortlived
+
+The shortlived profile is identical to the tlsserver profile, with one key distinction: the resulting certificate is only valid for 6ish days. This allows these certificates to qualify as "Short-Lived Subscriber Certificates" under the Baseline Requirements, which means they do not need to contain any revocation information. This makes the certificates even smaller, and removes any possibility of a client accidentally trusting a certificate after it has been revoked.
```

Comment:
The awkward thing about this is that the short-lived certs we're issuing will contain the OCSP URI. So we don't _have_ to contain revocation information, but we do.

Reply:
Yeah, I chose the phrasing "do not _need_ to contain" carefully. But I've rephrased slightly to be even more cagey about it.